components/org.wso2.carbon.identity.organization.management.service/pom.xml

```diff
@@ -84,6 +84,11 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.testutil</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
any reason to add this dependency?

Reply:
After the version upgrade one of the test case started to fail, to resolve it added this dependency.

Reply:
Ack.